blocks/header/headerutil.js

```diff
@@ -298,6 +298,109 @@ const processBondsType = (items) => {
   return processedData;
 };
 
+function loadStockFeed(gaTokenId) {
```

Comment:
@ravverma how about we call this function in delayed phase since it would require to load another script as well.

Reply:
header already load in `loadLazy`.

Reply:
I would still prefer loading an additional script in delayed phase as done here https://github.com/aemsites/icicidirect/pull/234/files#diff-02c3ef68f074c763af7a51cf09ad376dcfa1c9a905fd8c3679a92f1443722516R382
But since its not impacting the LHS score. I am fine with it.